src/vcpkg/commands.set-installed.cpp

```diff
@@ -277,6 +278,23 @@ namespace vcpkg
             }
         }
 
+        if (auto manifest = paths.get_manifest().get())
+        {
+            Json::Object manifest_info;
+            manifest_info.insert("manifest_path", Json::Value::string(manifest->path));
+
+            if (const auto installed_paths = paths.maybe_installed().get())
+            {
+                const auto json_file_path = installed_paths->vcpkg_dir() / "manifest_info.json";
+
+                const auto json_contents = Json::stringify(manifest_info, Json::JsonStyle::with_spaces(4));
```

Comment:
```suggestion
                const auto json_contents = Json::stringify(manifest_info);
```

I don't think there's a good reason for this file and only this file to use 4 spaces instead of 2?